sql/hive/src/test/resources/sqlgen/limit.sql

```diff
@@ -0,0 +1,4 @@
+-- This file is automatically generated by LogicalPlanToSQLSuite.
+SELECT * FROM (SELECT id FROM tbl LIMIT 2)
+--------------------------------------------------------------------------------
```

Comment:
Without this PR, this test case fails because it generate the following query. Note that the third line.
```
SELECT `gen_attr_0` AS `id`
FROM (SELECT `gen_attr_0`
             FROM SELECT `gen_attr_0`
                         FROM (SELECT `id` AS `gen_attr_0`, `name` AS `gen_attr_1`
                                     FROM `default`.`tbl`) AS gen_subquery_0 LIMIT 2) AS tbl
```